test/correctness/logical.cpp

```diff
@@ -94,10 +94,12 @@ int main(int argc, char **argv) {
         Target target = get_jit_target_from_environment();
 
         if (target.has_gpu_feature()) {
-            f.gpu_tile(x, y, 16, 16).vectorize(Var::gpu_threads(), 4);
+            f.gpu_tile(x, y, xi, yi, 16, 16).vectorize(xi, 4);
         } else if (target.features_any_of({Target::HVX_64, Target::HVX_128})) {
+assert(0);
```

Comment:
Because I was careless and didn't remove the scalpel from the patient before closing :-(